step-capstone/src/components/Event.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Box, Grid, Typography } from '@material-ui/core'
+
+/*
+Takes data in this form:
+        {
+          finalized: true,
+          type: "event",
+          title: "Visiting Sherlock",
+          location: "221b Baker Street, London",
+          timeStart: "8:30",
+          timeEnd: "10:30",
+          description: "Additional notes"
+        }
+
+*/
+export default function Flight( data ) {
+    return(
+        <Grid item>
+            <Typography variant="h6" gutterBottom>Title { data.title } to { data.title }</Typography>
+            <Typography variant="subtitle2" gutterBottom>Location: { data.location }</Typography>
+            <Typography variant="subtitle2" gutterBottom>Time start: { data.timeStart }</Typography>
+            <Typography variant="subtitle2" gutterBottom>Time end: { data.timeEnd }</Typography>
```

Comment:
I dont think we want all the descriptions of what each thing is to appear. so it should just be {data.location} for example, not Location: {data.location}. Also, I think Title is still left in a data format, and Time Start and Time End, if we do display them, should be done like start - end

Reply:
I agree with not labeling title and location, but labeling start and end times makes an equal amount of sense to me as start-end if you prefer that.

Reply:
I believe the date/time will come in as a Data object, so you would need to convert that to a string format (toString())